pkgs/development/libraries/libgssglue/default.nix

```
@@ -1,13 +1,36 @@
-{ lib, stdenv, fetchurl, libkrb5 }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoconf,
+  automake,
+  libtool,
+  libkrb5,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libgssglue";
-  version = "0.4";
+  version = "0.9.1";
```

Latest version is 0.9 according to upstream https://gitlab.com/gsasl/libgssglue/-/releases

```suggestion
  version = "0.9";
```